src/routes/(app)/components/CreateLink/CreateLink.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+  import { getContextClient } from '@urql/svelte';
+  import { newForm } from '@whizzes/svelte-forms';
+  import * as Yup from 'yup';
+  import { page } from '$app/stores';
+
+  import Button from '$lib/components/Button.svelte';
+  import Card from '$lib/components/Card.svelte';
+  import TextField from '$lib/components/TextField.svelte';
+  import notification from '$lib/stores/notification';
+  import { LinkCreateDocument } from '$lib/graphql/schema';
+
+  const urqlClient = getContextClient();
+
+  const { handleSubmit, values, errors, isSubmitting } = newForm({
+    initialValues: {
+      url: '',
+      customHash: ''
+    },
+    validationSchema: Yup.object({
+      url: Yup.string().required(),
+      hash: Yup.string()
+    }),
+    onSubmit: async ({ url, customHash }) => {
+      const payload: { url: string; customHash?: string } = {
+        url
+      };
+
+      if (customHash.length) payload.customHash = customHash;
+
+      const result = await urqlClient
+        .mutation(
+          LinkCreateDocument,
+          {
+            input: payload
+          },
+          {
+            fetchOptions: {
+              headers: { authorization: `JWT ${$page.data.accessToken}` }
+            }
+          }
```

Comment:
I think we can improve this a bit later by using the Authorization Extension from URQL

Reply:
How we can use the authorization extension?

Reply:
The Authentication Exchange is part of the URQL ecosystem, you can read more about it here:

https://formidable.com/open-source/urql/docs/api/auth-exchange/

But lets address that in a separate PR, after merging this.